code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -49,7 +49,7 @@
 					new /obj/item/weapon/gun/projectile/pistol(get_turf(H))
 					new /obj/item/gun_part/silencer(get_turf(H))
 				if("cannon")
-					new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
+					new /obj/item/weapon/gun/energy/laser/cannon(get_turf(H))
```

Comment:
Seriously calm the fuck down.

First of all, this is a good change by @Sprok0.

Second of all: I think the poll was pretty damn descriptive without having to open the PR itself.

Christ man if you are against this then actually get some arguments together instead of flinging shit at literally everything.